usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf

```diff
@@ -1,48 +1,52 @@
 # Begin example setup for SAP HANA on UEFI capable systems.
-# You must adapt "your.NFS.server.IP/path/to/your/rear/backup" at BACKUP_URL.
-# You may activate SSH_ROOT_PASSWORD and adapt the 'password_on_the_rear_recovery_system'.
-# For basic information see the SLE11 and SLE12 manuals.
+
+# You must adapt "your.NFS.server.IP/path/to/your/rear/backup" at BACKUP_URL and also
+# the HANA related volume groups and mount points at EXCLUDE_VG and BACKUP_PROG_EXCLUDE.
+# As needed you may activate SECURE_BOOT_BOOTLOADER and USE_DHCLIENT
+# and also SSH_ROOT_PASSWORD with a 'password_on_the_rear_recovery_system'.
```

Comment:
Is setting an SSH password required for SAP HANA recovery? Otherwise I'd prefer leaving this out to not lead users into insecure configurations